examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
Would it not be possible to put this rather simple code in a function that is templated on the grid type and returns the material law (taking grid, deck and eclipsestate as members)? That could then be used in four different spots in this module and also in opm-polymer instead of copying.